migrations/witnet-queries.json

```diff
@@ -339,7 +342,7 @@
         "decimals": 6
     },
     "StellaUsdtPrice": {
-        "bytecode": "0x0ae10212570801123a68747470733a2f2f6170692e686f746269742e696f2f6170692f76312f6d61726b65742e6c6173743f6d61726b65743d5354454c4c41555344541a1784187782186466726573756c748218571a000f4240185b12e7010803123e68747470733a2f2f6170692e74686567726170682e636f6d2f7375626772617068732f6e616d652f7374656c6c61737761702f7374656c6c612d737761701a2c861877821866646461746182186664706169728218646b746f6b656e3150726963658218571a000f4240185b22757b227175657279223a227b5c6e20202020202070616972202869643a205c223078383165313161393337343033336431316363376537343835613731393261653337643037393564365c2229207b5c6e2020202020202020746f6b656e3150726963655c6e2020202020207d5c6e202020207d227d1a0d0a0908051205fa3fc000001003220d0a0908051205fa40200000100310c0843d180a20c0843d28333080e497d012",
+        "bytecode": "0x0a880212e7010803123e68747470733a2f2f6170692e74686567726170682e636f6d2f7375626772617068732f6e616d652f7374656c6c61737761702f7374656c6c612d737761701a2c861877821866646461746182186664706169728218646b746f6b656e3150726963658218571a000f4240185b22757b227175657279223a227b5c6e20202020202070616972202869643a205c223078383165313161393337343033336431316363376537343835613731393261653337643037393564365c2229207b5c6e2020202020202020746f6b656e3150726963655c6e2020202020207d5c6e202020207d227d1a0d0a0908051205fa3fc000001003220d0a0908051205fa40200000100310c0843d180a20c0843d28333080e497d012",
```

Comment:
```
$ npx -p witnet-requests@0.9.3 witnet-toolkit try-query --hex  0x0a880212e7010803123e68747470733a2f2f6170692e74686567726170682e636f6d2f7375626772617068732f6e616d652f7374656c6c61737761702f7374656c6c612d737761701a2c861877821866646461746182186664706169728218646b746f6b656e3150726963658218571a000f4240185b22757b227175657279223a227b5c6e20202020202070616972202869643a205c223078383165313161393337343033336431316363376537343835613731393261653337643037393564365c2229207b5c6e2020202020202020746f6b656e3150726963655c6e2020202020207d5c6e202020207d227d1a0d0a0908051205fa3fc000001003220d0a0908051205fa40200000100310c0843d180a20c0843d28333080e497d012
╔════════════════════════════════════════════╗
║ Witnet query local execution report        ║
╚╤═══════════════════════════════════════════╝
 │
 │  ┌────────────────────────────────────────────────┐
 ├──┤ Retrieval stage                                │
 │  ├────────────────────────────────────────────────┤
 │  │ Number of retrieved data sources: 1            │
 │  └┬───────────────────────────────────────────────┘
 │   │
 │   └─[ Source #0 (api.thegraph.com) ]
 |      Method: HTTP-POST
 |      Complete URL: https://api.thegraph.com/subgraphs/name/stellaswap/stella-swap
 |      Body: {"query":"{\n      pair (id: \"0x81e11a9374033d11cc7e7485a7192ae37d0795d6\") {\n        token1Price\n      }\n    }"}
 |      Number of executed operators: 6
 |      Execution time: 0.03128 ms
 |      Execution trace:
 │        [0] HTTP-POST -> String: "{\"data\":{\"pair\":{\"token1Price\":\"0.1265483572641528452865987763929401\"}}}"
 │        [1] .parseJSONMap() -> Map: {"data":{"RadonMap":{"pair":{"RadonMap":{"token1Price":{"RadonString":"0.1265483572641528452865987763929401"}}}}}}
 │        [2] .getMap(data) -> Map: {"pair":{"RadonMap":{"token1Price":{"RadonString":"0.1265483572641528452865987763929401"}}}}
 │        [3] .getMap(pair) -> Map: {"token1Price":{"RadonString":"0.1265483572641528452865987763929401"}}
 │        [4] .getFloat(token1Price) -> Float: 0.12654835726415284
 │        [5] .multiply(1000000) -> Float: 126548.35726415284
 │        [6] .round() -> Integer: 126548
 |      Result: Integer: 126548
 │
 │  ┌────────────────────────────────────────────────┐
 ├──┤ Aggregation stage                              │
 │  ├────────────────────────────────────────────────┤
 │  │ Execution time: 0.0051 ms                      │
 │  │ Result is Integer: 126548                      │
 │  └────────────────────────────────────────────────┘
 │  
 │  ┌────────────────────────────────────────────────┐
 └──┤ Tally stage                                    │
    ├────────────────────────────────────────────────┤
    │ Execution time: 0.00116 ms                     │
    │ Result is Integer: 126548                      │
    └────────────────────────────────────────────────┘
```

Reply:
This one can be made cheaper after https://github.com/witnet/witnet-requests-js/pull/77

Reply:
ok, i'll then wait until witnet/witnet-requests-js#77 gets merged and tagged.

Reply:
It's merged and tagged as 0.9.4 now.